whois-scheduler/src/main/java/net/ripe/db/whois/scheduler/task/grs/LacnicGrsSource.java

```diff
@@ -46,6 +49,8 @@ class LacnicGrsSource extends GrsSource {
     private final String userId;
     private final String password;
 
+    private final String irrDownload;
+
     private Client client;
```

Comment:
why not final?

Reply:
It is final?

Reply:
Did you mean to comment on the `client` field?

Reply:
I made the `client` field final